app/controllers/api/providers_controller.rb

```diff
@@ -107,6 +107,11 @@ def destroy_provider(provider)
     end
 
     def update_provider_authentication(provider, data)
+      connection_configurations = data[CONNECTION_ATTR]
+      unless connection_configurations.blank?
+        provider.authentication_check_types_queue(provider.default_authentication, :save => true)
```

Comment:
`provider.authentication_check_types_queue(provider.default_authentication, :save => true) unless data[CONNECTION_ATTR].blank?`

@simon3z 
this line is ok, but rubocop say it's too long :-( any idea ?